lib/guardian_db/token.ex

```diff
@@ -57,9 +55,19 @@ defmodule GuardianDb.Token do
   """
   def purge_expired_tokens! do
     timestamp = Guardian.timestamp()
-    query = from token in Token,
-              where: token.exp < ^timestamp
 
-    GuardianDb.repo.delete_all(query)
+    query_schema()
+    |> Query.where([token], token.exp < ^timestamp)
```

Comment:
You imported `where/2` above, is it necessary to refer to it as `Query.where/2`?

Reply:
@doomspork I removed the `import` but my compilation failed for some reason, so I had to add it back in, no clue why